index.html

```diff
@@ -24,7 +24,13 @@ <h1>Pick a theme</h1>
   <script>
 
     const pressedButtonSelector = '[data-theme][aria-pressed="true"]';
-    const defaultTheme = 'green';
+    const prefersColorMediaQuery = window.matchMedia("(prefers-color-scheme: dark)");
+    const themes = {
+      saved: localStorage.getItem('selected-theme'),
+      dark: 'blue', // Theme to load when prefers-colors-scheme = light
+      light: 'pink', // Theme to load when prefers-colors-scheme = dark
+      default: 'orange' // Theme to load when prefers-colors-scheme isn’t supported
```

Comment:
Not sure if these comments here are helpful/needed

Reply:
I like the comments 👍🏻